libmodernize/main.py

```diff
@@ -28,6 +29,14 @@ def format_usage(usage):
     return usage
 
 def main(args=None):
+    old_os_linesep = os.linesep
+    try:
+        return actual_main(args)
+    finally:
+        if sys.version_info < (3, 0):
+            os.linesep = old_os_linesep
```

Comment:
If we are on Python 3, the monkeypatch is never removed, which will make the tests do different things depending on what order they run in.